File: Unidad 5/C#/P8_U5.cs
using System;

class P8_U5
{
    static void Main()
    {
        int num_dad = 2; 
        int resultado = Lanzar_dad(num_dad);

        Console.WriteLine("Resultado del lanzamiento: " + r);
    }

    static int Lanzar_dad(int num_dad)
    {
        Random random = new Random();
        int r = 0;

        for (int i = 0; i < num_dad; i++)
        {
            int valor_dad = random.Next(1, 7);
            Console.WriteLine($"Dado {i + 1}: {valor_dad}");

            r += valor_dad;
        }

        return r;
    }
}

File: Unidad 5/JAVA/P8_U5.java
import java.util.Scanner;
public class P8_U5
{
	public static void main(String[] args) {
        int numero_Dados = 2; 
        int resultado = lanzar_Dados(numero_Dados);

        System.out.println("Resultado del lanzamiento: " + resultado);
    }

    public static int lanzar_Dados(int numero_Dados) {
        Random random = new Random();
        int resultado = 0;

        for (int i = 0; i < numero_Dados; i++) {

            int valor_Dado = random.nextInt(6) + 1;
            System.out.println("Dado " + (i + 1) + ": " + valor_Dado);

            resultado += valor_Dado;
        }

        return resultado;
    }
}